src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -60,9 +60,12 @@ public static partial class AppContext
         public static string BaseDirectory { get { return default(string); } }
         public static void SetSwitch(string switchName, bool isEnabled) { }
         public static event System.UnhandledExceptionEventHandler UnhandledException { add { } remove { } }
+        public static event System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs> FirstChanceException { add { } remove { } }
```

Comment:
These additions should be done as netcoreapp1.1 specific. It may be better to do them separately, and have proper api-addition issue for them - I do not think you need to add them to the public contract for the appdomain API work (adding them to the implementation is fine).